### PR TITLE
AcceptOfferController/Logging #102

### PR DESCRIPTION
**Pull request by raghavancgn**

### CODE REVIEW

File: src/main/java/com/kodekonveyor/work_request/offer/AcceptOfferController.java [hunk 1]
```diff
@@ -21,10 +22,12 @@
...
+  @Autowired
+  LoggerService loggerService;
 
   @PutMapping("/accept/{offerId}")
   public WorkRequestDTO call(final long offerId) {
-
+    loggerService.call(OfferConstants.ACCEPT_OFFER_CONTROLLER_IS_STARTED);
     final OfferEntity offerEntity =
         offerEntityRepository.findById(offerId).get();
     final WorkRequestEntity workRequest = offerEntity.getWorkRequest();
```
Comment (szirbucz) [inline src/main/java/com/kodekonveyor/work_request/offer/AcceptOfferController.java:33]: Please log before line 36 for success case.

**Comment by szirbucz** [comment]
Please show some activity, otherwise this task will be ungrabbed from you in 2 days.

File: .settings/org.eclipse.jdt.core.prefs [hunk 2]
```diff
@@ -1,15 +1,17 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
-org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.methodParameters=generate
```
Comment (magwas) [inline .settings/org.eclipse.jdt.core.prefs:3]: please revert this file

File: src/test/java/com/kodekonveyor/work_request/offer/AcceptOfferControllerLoggingTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
+//Created by Raghavan
```
[the remark this answers is not in the document]
Comment (raghavancgn) [reply src/test/java/com/kodekonveyor/work_request/offer/AcceptOfferControllerLoggingTest.java:1]: removed the comments

**Pull request merged**